Monika After Story/game/script-topics.rpy

```diff
@@ -15749,3 +15749,54 @@ label monika_literature_value:
     m 1eku "Honestly, I think if more people valued books and poems a little more, the world would be a much better place."
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+    
+# Do you like nature?
+default persistent._mas_pm_likes_nature = None
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_nature",category=['nature', 'you'],prompt="The outdoors",random=True))
+ 
+label monika_nature:
+    m 2esd "Hey, [player]..."
+    $ _history_list.pop()
+    menu:
+        m "Do you like nature{fast}?"
+        "Yes.":
+            $ persistent._mas_pm_likes_nature = True
+            m 2sub "Really? That's wonderful!"
+            m 3wud "You know, I think nature is something we should cherish."
+            m 3eua "Not only is it beautiful, but it also helps humanity out."
+            m 4esc "Insects pollinate our crops, trees give us wood and shade, pets offer us companionship..."
+            m 3esd "Most of all, autotrophs such as plants and algae produce food and oxygen; without them, most life on Earth wouldn't exist."
+            m 1esa "Because of that, I think it's only fair that we give something back to nature since it does so much for us."
+            m 7hub "So, here's Monika's Green Tip of the Day!"
+            m 7eud "Sometimes, people hesitate to go green because they're worried that it's too expensive."
```

Comment:
I agree that this is where Monika would have more of a serious face. I still think I prefer this one though.

![1](https://user-images.githubusercontent.com/45746485/93428747-46980a80-f8c0-11ea-9883-91d2383ec17a.png)

Reply:
Good idea, I really like the sideways look here.